python/feast_spark/client.py

```diff
@@ -93,6 +100,12 @@ def _job_service(self):
             self._job_service_service_stub = JobServiceStub(channel)
         return self._job_service_service_stub
 
+    def is_whitelisted(self, project: str):
```

Comment:
I think this is purely JobService related and shouldn't be part of the SparkClient